site-src/guides/tls.md

```diff
@@ -77,6 +85,12 @@ before an HTTP request is sent from the client.
 [TLS Certificate in Route](#tls-certificate-in-route) provides an example
 of how this feature can be used.
 
+Also, as mentioned above, the Route Kind (`HTTPRoute`, `TLSRoute`, `TCPRoute`) is dependant on the protocol on the listener level. Listener Protocol `HTTPS` or `HTTP` allows to bind against `HTTPRoute` as the TLS Termination is done at the listener level and thus, only http information are used for routing.
```

Comment:
Also one more tiny nit - do you mind wrapping these lines at 80 characters?

Reply:
Done